packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasEstimates.ts

```diff
@@ -30,7 +33,17 @@ async function fetcher([
 
   return await bridgeTransferStarter.transferEstimateGas({
     amount,
-    signer
+    signer,
+    onError(error) {
+      const code = (error as { code: string }).code
+      if (
+        code === 'INSUFFICIENT_FUNDS' ||
+        (code === 'UNPREDICTABLE_GAS_LIMIT' && amount.gte(balance))
+      ) {
+        // If amount is equal or above balance's user, we should not have run the gas estimate
```

Comment:
`        // If amount is equal or above user's balance, we should not have run the gas estimate
`

Reply:
Also should we also not run it in case of equal balance?